.cirrus.yml

```
@@ -2,75 +2,92 @@ Lint_task:
-    image: mojave-base
-  env:
-    PATH: ${HOME}/.pyenv/shims:${PATH}
-    matrix:
-      - PYTHON: 3.6.8
-      - PYTHON: 3.7.2
-  install_script:
-    # Per the pyenv homebrew recommendations.
-    # https://github.com/pyenv/pyenv/wiki#suggested-build-environment
-    - sudo installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg -target /
-    - brew install openssl readline pyenv
-    - pyenv install ${PYTHON}
-    - pyenv global ${PYTHON}
-    - pyenv rehash
-    - pip install -e .[testing]
-  script:
-    - python3 --version
-    - python3 -m pytest --verbose
+# macOS_task:
+#   osx_instance:
+#     image: mojave-base
+#   env:
+#     PATH: ${HOME}/.pyenv/shims:${PATH}
+#     matrix:
+#       - PYTHON: 3.6.8
+#       - PYTHON: 3.7.2
+#   install_script:
+#     # Per the pyenv homebrew recommendations.
+#     # https://github.com/pyenv/pyenv/wiki#suggested-build-environment
+#     - sudo installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg -target /
+#     - brew install openssl readline pyenv
+#     - pyenv install ${PYTHON}
+#     - pyenv global ${PYTHON}
+#     - pyenv rehash
+#     - pip install -e .[testing]
+#   script:
+#     - python3 --version
+#     - python3 -m pytest --verbose
 
-FreeBSD_task:
-  freebsd_instance:
-    image: freebsd-12-0-release-amd64
-  env:
-    matrix:
-      - PYTHON: 3.6
-      - PYTHON: 3.7
-  install_script:
-    - PY=`echo $PYTHON | tr -d '.'`
-    - pkg install -y python${PY} py${PY}-setuptools
-    - python${PYTHON} -m ensurepip
-    - python${PYTHON} -m pip install -e .[testing]
-  script:
-    - python${PYTHON} --version
-    - python${PYTHON} -m pytest --verbose
+# FreeBSD_task:
+#   freebsd_instance:
+#     image: freebsd-12-0-release-amd64
+#   env:
+#     matrix:
+#       - PYTHON: 3.6
+#       - PYTHON: 3.7
+#   install_script:
+#     - PY=`echo $PYTHON | tr -d '.'`
+#     - pkg install -y python${PY} py${PY}-setuptools
+#     - python${PYTHON} -m ensurepip
+#     - python${PYTHON} -m pip install -e .[testing]
+#   script:
+#     - python${PYTHON} --version
+#     - python${PYTHON} -m pytest --verbose
 
-Windows_task:
-  allow_failures: $CIRRUS_TASK_NAME =~ '.*-rc-.*'
-  windows_container:
-    os_version: 2019
-    matrix:
-      - image: python:3.6-windowsservercore-1809
-      - image: python:3.7-windowsservercore-1809
-      - image: python:3.8-rc-windowsservercore-1809
+# Windows_task:
+#   allow_failures: $CIRRUS_TASK_NAME =~ '.*-rc-.*'
+#   windows_container:
+#     os_version: 2019
+#     matrix:
+#       - image: python:3.6-windowsservercore-1809
+#       - image: python:3.7-windowsservercore-1809
+#       - image: python:3.8-rc-windowsservercore-1809
 
+#   install_script:
+#     - C:\Python\python.exe -m pip install -e .[testing]
+#   script:
+#     - C:\Python\python.exe --version
+#     - C:\Python\python.exe -m pytest --verbose
+
+repos_task:
+  container:
+    image: debian:slim
```

There is no just `slim` tag. You might want to use something like:

```suggestion
    image: debian:stable-sli
```

---

Oh, thanks! I loose track of these things.